client/src/lit/interactive-example/tabs.wrapper.scss

```
@@ -0,0 +1,14 @@
+:host {
+  display: flex;
+  flex-direction: column;
+  overflow: hidden;
+}
+
+#tablist {
+  background: var(--background-secondary);
+  border-bottom: 1px solid var(--border-secondary);
+  display: flex;
+  flex-shrink: 0;
```

Nit: Since `flex-grow` defaults to 0 (unless set elsewhere I don't see), we can use the short-hand for clarity (!?).

```suggestion
  flex: 0;
```

---

I think it would have to be `flex: 0 0;`, and at least to me using flex-shrink directly is clearer: I think "this flex item can't shrink" rather than "wait what order are the arguments in flex again"